README.md

```diff
@@ -16,28 +16,29 @@ workshop.
 
 ### Background slides:
 
-The accompanying slides for each module are linked below and briefly introduce
-the concepts underlying each module.
+The accompanying slides for each module are linked below and briefly introduce the concepts underlying each module.
 
 - [intro-to-R-tidyverse](https://drive.google.com/a/ccdatalab.org/file/d/11GXEddKwUBan1Z-NF_UM2HecckpODLKM/view?usp=sharing)
 - [RNA-seq](https://drive.google.com/a/ccdatalab.org/file/d/1A9gNDIuD_c3ppF2k6vY3b0VgSKZjchzp/view?usp=sharing)
 - [scRNA-seq](https://drive.google.com/a/ccdatalab.org/file/d/186niFprBKICNsF53WpIhKbiIMLawu-ms/view?usp=sharing)
 - [machine-learning](https://drive.google.com/a/ccdatalab.org/file/d/1tmX8sFDmnPpkRdkWQm-v3vtdGr6YWy-j/view?usp=sharing)
+- Advanced scRNA-seq
+  - [Differential expression](https://drive.google.com/file/d/1nT1AxEYTMyizXvlrpZCrk8HfdJH8hcWN/view?usp=sharing)
```

Comment:
Intro: https://docs.google.com/presentation/d/1qjmx_C0P5feswxXvn6ezCRE5OObtB9iftQvI3ceUAa0/edit?usp=sharing

Cell typing (placeholder): https://docs.google.com/presentation/d/16ivLdwm17jqHZKOnRGVjigIMAZtr9Xpc5kjspn9mIhQ/edit?usp=sharing

Integration: https://docs.google.com/presentation/d/1l1Nmj_OADnfZ6J8DaIj7ncetCo_cbLvOI-2xtpW_FrI/edit?usp=sharing

Ah! I just realized that the link you added (and the links to previous slides) are to PDF versions. I actually think people may like to get them as google slides, and that should allow us easier updating. (Many of the other slides are out of date to what we do now). Can we update all of them?

Part of my reasoning here is also that I want to freeze this repo today, and my slides aren't done 🙀 

We might also change all the slide and cheatsheet links while we are at it to be text rather than hyphen-separated?

Reply:
Yea I just used PDF since all the other links were PDF, but I went ahead and changed them all to links to the slides including the older ones. But I didn't change the order for any of the old ones or add additional ones. 

I also removed the hyphens and changed everything to sentence case. The other thing I want to note before we release is that we don't have the readme for `scRNA-seq-advanced` filled out yet. Do you want me to add that in in this PR?

Reply:
> Do you want me to add that in in this PR?

I personally will not argue with this!

Reply:
Okay I'm going to go ahead and add it in.